backend/src/middleware/hashtags/extractHashtags.spec.js

```diff
@@ -8,9 +8,24 @@ describe('extractHashtags', () => {
   })
 
   describe('searches through links', () => {
-    it('finds links with and without ".hashtag" class and extracts Hashtag names', () => {
-      const content =
-        '<p><a class="hashtag" href="/search/hashtag/Elections">#Elections</a><a href="/search/hashtag/Democracy">#Democracy</a></p>'
+    it('without `class="hashtag"` but `data-hashtag-id="something"`, and extracts the Hashtag to make a Hashtag link', () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-17T07:22:15Z" title="Tuesday, September 17th 2019, 9:22:15 am +02:00">Sep 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -8,9 +8,24 @@ describe('extractHashtags', () => {
   })
 
   describe('searches through links', () => {
-    it('finds links with and without ".hashtag" class and extracts Hashtag names', () => {
-      const content =
-        '<p><a class="hashtag" href="/search/hashtag/Elections">#Elections</a><a href="/search/hashtag/Democracy">#Democracy</a></p>'
+    it('not `class="hashtag"` but `data-hashtag-id="something"` makes a link a hashtag link', () => {
```


This sounds nice, but is not true in real.
Because the `data-hashtag-id` doesn’t make it in the editor again.
On reedit it is no hashtag link anymore in this sense.